src/utils/logs.ts

```diff
@@ -0,0 +1,37 @@
+interface GetLogsLinkProps {
+    dbName: string;
+    logging: string;
+}
+
+export type GetLogsLink = (props: GetLogsLinkProps) => string;
+
+export function getLogsLink({dbName, logging}: GetLogsLinkProps): string {
+    try {
+        const data = JSON.parse(logging);
+
+        if (typeof data === 'object' && 'url' in data) {
+            const logUrl = data.url;
+            if (!logUrl) {
+                return '';
+            }
+
+            const url = new URL(logUrl);
+
+            const queryParam = url.searchParams.get('query');
+            if (queryParam) {
+                const decodedQuery = decodeURIComponent(queryParam);
+
+                const updatedQuery = decodedQuery.replace(/\{([^}]*)\}/, (_match, contents) => {
```

Comment:
Please, explain this with some example. Like before => after. In code comments, not here

Reply:
made code self-explanatory